processing/src/main/java/org/apache/druid/segment/QueryableIndexCursorHolder.java

```diff
@@ -112,6 +113,7 @@ public QueryableIndexCursorHolder(
             Cursors.getTimeOrdering(ordering),
             interval,
             filter,
+            cursorBuildSpec.getQueryContext().getBoolean(QueryContexts.CURSOR_AUTO_ARRANGE_FILTERS, false),
```

Comment:
we should probably default this to true, or are you planning to wait until we have added the cost computations?